packages/batch-tx-handler/src/l1-batch-handler/utils.ts

```diff
@@ -147,7 +147,7 @@ export const getRawData = async (sequencerTx: string): Promise<[Uint8Array, BigN
   if (!tx || !txReceipt || !txReceipt.status) {
     throw new Error('No such a l1 transaction or transaction reverted');
   }
-  if (tx.to.toLowerCase() !== l2Network.ethBridge.sequencerInbox.toLowerCase()) {
+  if (tx.to!.toLowerCase() !== l2Network.ethBridge.sequencerInbox.toLowerCase()) {
```

Comment:
there's always a possibility where `tx` returns without a valid `to`
so you need to add a check for that before this line, then you don't need to use the non-null assertion operator `!`

i.e.
```typescript
if (!tx.to) {
  throw new Error('Transaction does not contain recipient address');
}
```